emission/storage/timeseries/builtin_timeseries.py

```diff
@@ -23,6 +23,9 @@
 INVALID_QUERY = {'metadata.key': 'invalid'}
 
 class BuiltinTimeSeries(esta.TimeSeries):
+
+    entryList=[]
+
```

Comment:
@humbleOldSage this is wrong. There is not only one set of entries in the database. Please read and understand the data model from chapter 5 of my thesis.

Reply:
Figured this was not necessary